.github/CODEOWNERS

```diff
@@ -5,7 +5,7 @@
 /utils/build/docker/golang*/ @DataDog/dd-trace-go-guild @DataDog/system-tests-core
 /utils/build/docker/java*/ @DataDog/apm-java @DataDog/asm-java @DataDog/system-tests-core
 /utils/build/docker/java_otel/ @DataDog/opentelemetry @DataDog/system-tests-core
-/utils/build/docker/nodejs*/ @DataDog/apm-js @DataDog/asm-js @DataDog/system-tests-core
+/utils/build/docker/nodejs*/ @DataDog/dd-trace-js @DataDog/system-tests-core
```

Comment:
@DataDog/asm-js and @DataDog/apm-js are part of @DataDog/dd-trace-js ? cc @simon-id